src/layouts/index.html

```diff
@@ -140,6 +140,6 @@ <h1 class="title">Lightning Web Components Recipes</h1>
             </ul>
         </header>
 
-        <div id="main"></div>
+        <div id="main">{{{body}}} {{{lwr_resources}}}</div>
```

Comment:
`{{{ lwr_resources }}}` should be a child element to the `body`.

Reply:
Fixed it in https://github.com/trailheadapps/lwc-recipes-oss/pull/274

Reply:
With the new LWR, how should we import "@lwc/synthetic-shadow"?

Reply:
In lwr.config.json, you'll have to add `"bootstrap": { "syntheticShadow": true}` as an option in the route for which you want synthetic shadow. For example - 
```
"routes": [
        {
            "id": "home",
            "path": "/",
            "rootComponent": "ui/app",
            "layoutTemplate": "$layoutsDir/index.html"
            "bootstrap": {
                "syntheticShadow": true
            }
        }
    ]
```